config/openshift/v4_13/translate_test.go

```diff
@@ -490,8 +403,7 @@ func TestValidateSupport(t *testing.T) {
 							},
 							Links: []base.Link{
 								{
-									Path:   "/l",
-									Target: util.StrToPtr("/t"),
```

Comment:
Looks like this got dropped?

Reply:
Yeah, I accidentally dropped it. Thank you for catching that.